.github/workflows/format-code.yml

```diff
@@ -9,15 +9,10 @@ jobs:
     name: 'Format code'
     runs-on: ubuntu-latest
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/format')
-    steps:
-      - name: 'Check fork status'
-        id: fork_check
-        run: |
-          IS_FORK="$(jq '.pull_request.head.repo.fork' "$GITHUB_EVENT_PATH")"
-          echo "::set-output name=fork::$IS_FORK"
-          
+    steps:          
       - name: 'Setup SSH deploy key'
-        if: steps.fork_check.outputs.fork == "false"
+        # This will need to be adjusted when PRs are run against forks of the v3 repo
+        if: github.repository_owner == 'exercism'
```

Comment:
I like this new way of checking for forks! Much simpler.

Reply:
Unfortunately it doesn't work 😬
Since the action always runs in the v3 repo, it would always be `'exercism'`

Reply:
Ah, too bad! Would have been nice :)